configuration/pih/htmlforms/hiv/zl/hiv-dispensing.xml

```diff
@@ -185,11 +196,7 @@
 
 
         var validateNextDispensingDate = function() {
-            var currentEncounterDate = new Date();
-            var encounterDate = '<lookup expression="encounter.getEncounterDatetime().getTime()"/>';
-            if (typeof encounterDate !== "undefined" &amp;&amp; encounterDate !== null &amp;&amp; (encounterDate.length > 0)) {
-                currentEncounterDate = new Date(+encounterDate);
-            }
+            var currentEncounterDate = new Date(getField('encounterDate.value').datepicker('getDate'));
```

Comment:
Just new Date(getField('encounterDate.value'))" isn't enough?

Again, just curious, not a big deal if it works as-is...